apps/i18n/common/en_us.json

```diff
@@ -974,30 +974,32 @@
   "projectTypeApplab": "App Lab",
   "projectTypeArtist": "Artist",
   "projectTypeArtistPreReader": "Artist (Pre-reader)",
-  "projectTypeGamelab": "Game Lab",
-  "projectTypeGamelabBeta": "Game Lab (beta)",
-  "projectTypePlaylab": "Play Lab",
```

Comment:
@Erin007 Are these project types obsolete?

Reply:
No - well, maybe Game Lab beta but deleting that isn't part of this change.  I didn't delete these strings; I re-ordered them so the projectTypes are alphabetized. The strings show up green on the "new" side of the diff a few lines later.